### PR TITLE
Feature/client side fixes

### PR DESCRIPTION
**Pull request by sync**
Was playing with the select and noticed that we weren't including NextScript in the _document. Was this choice by design ? 

Also once I included the script I realised that Apollo wasn't instantiated properly on the client side. And also react was throwing some warnings about the select

### CODE REVIEW

File: www/next.config.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const nextConfig = {
     googleAnalytics: isDev ? "" : "UA-45226320-5",
     isDev
   },
-  dontAutoRegisterSW: true,
+  dontAutoRegisterSw: true,
   workboxOpts: {
```
Comment (sync) [inline www/next.config.js:14]: also found a spelling mistake

File: www/pages/_document.tsx [hunk 2]
```diff
@@ -62,6 +62,7 @@ export default class MyDocument extends Document<Props> {
         </head>
         <body>
           <Main />
+          <NextScript />
```
Comment (sync) [inline www/pages/_document.tsx:66]: with this included, the react app will properly boot on the client, for example in dev mode the React tab was never shown inside the chrome debugger

**Comment by sync** [comment]
Deployed here: https://trends-lyg6y9977.now.sh/

File: www/pages/_document.tsx (hunk 2 above)
Comment (hanford) [reply www/pages/_document.tsx:66]: ahh, omitting this is intentional. We don't actually need any client side javascript other than https://github.com/hanford/trends/blob/e141ad37e42ec8aaa0b3aa02d7f43f28a0559867/www/pages/_document.tsx#L98

We're only using react server side to generate the templates, and there is only a few lines of client side javascript aka no react

Comment (hanford) [reply www/pages/_document.tsx:66]: including react would make a perfect performance audit much harder, potentially  impossible

Comment (hanford) [reply www/pages/_document.tsx:66]: the only client side javascript registered a event listener to submit a form when changing input values, and registers a service worker for offline :)

**Comment by sync** [comment]
@hanford yes we loose one point on the audit

screen shot 2018-12-14 at 9 15 57 am

File: www/pages/_document.tsx (hunk 2 above)
Comment (sync) [reply www/pages/_document.tsx:66]: cool if it's intentional I am happy to close this PR, I was playing with the router and just rerunning the query on the client side when changing the language from the input and realised React wasn't properly booted

**Comment by hanford** [comment]
@sync one of the things I really wanted in this project was to build an entirely react/javascript app with very little to 0 client side react. I thought that was really powerful and an interesting use case of Next.js that hadn't really been explored so I'd like to keep ^ the case if possible.

**Comment by sync** [comment]
Yes absolutely I understand. I'll close this PR because it makes more sense not to have it included.

**Comment by sync** [comment]
thanks for the clarification :-)

**Pull request closed without merging**